modules/tiles/src/tileset/tileset-3d.ts

```diff
@@ -66,8 +66,12 @@ export type Tileset3DProps = {
   attributions?: string[];
   headers?: any;
   loadTiles?: boolean;
-  fetchOptions?: {[key: string]: any};
+  loadOptions?: {[key: string]: any};
+  updateTransforms?: boolean;
+  maxRequests?: number;
```

Comment:
We could also supply a `requestSchedulerOptions` object. It would avoid creating one more mapping of options and the documentation could be unified.

Reply:
But the only relevant request scheduler option is `maxRequests`.
This line actually slipped in here from the traversal PR which I still need to rebase.